tests/framework/extensions/clusters/clusterconfig.go

```diff
@@ -29,6 +29,7 @@ type ClusterConfig struct {
 	Registries           *provisioningInput.Registries            `json:"registries" yaml:"registries"`
 	UpgradeStrategy      *rkev1.ClusterUpgradeStrategy            `json:"upgradeStrategy" yaml:"upgradeStrategy"`
 	Advanced             *provisioningInput.Advanced              `json:"advanced" yaml:"advanced"`
+	SSHTests             []string                                 `json:"sshTests" yaml:"sshTests"`
```

Comment:
I don't think you pushed up your changes, ssh tests is still here.

Reply:
Don't I still need this? I reference SSHTests in verify.go from here

Reply:
Then what's the point having it be its own entry?

Reply:
I agree, this shouldn't be necessary since sshTests doesn't live in the provisioning config

Reply:
We decided to keep it here Caleb, we would need to add another parameter if we want ssh tests in its own entry. I don't think it's worth it then